src/drive/mobile/lib/reporter.js

```diff
@@ -1,14 +1,19 @@
 /* global __SENTRY_TOKEN__, __DEVELOPMENT__ */
 import Raven from 'raven-js'
 
-const PROD_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
-const DEV_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
-export const ANALYTICS_URL = __DEVELOPMENT__
-  ? DEV_ANALYTICS_URL
-  : PROD_ANALYTICS_URL
-
 let isEnabled = false
 
+const getAnalyticsUrl = () => {
+  if (window.__SENTRY_TOKEN__ === undefined) {
```

Comment:
Do we need to use `window` here? `__SENTRY_TOKEN__ ` is provided as a global variable by webpack, but it's not *really* related to window.